Sources/Apodini/Semantic Model Builder/gRPC/GRPCService/GRPCServiceClientStreaming.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  GRPCServiceClientStreaming.swift
+//  
+//
+//  Created by Moritz Schüll on 07.01.21.
+//
+
+import Foundation
+@_implementationOnly import Vapor
+
+// MARK: Client streaming request handler
+extension GRPCService {
+    func createClientStreamingHandler<C: ConnectionContext>(context: C)
+    -> (Vapor.Request) -> EventLoopFuture<Vapor.Response> where C.Exporter == GRPCInterfaceExporter {
+        { (request: Vapor.Request) in
+            if !self.checkContentType(request: request) {
+                return request.eventLoop.makeFailedFuture(GRPCError.unsupportedContentType(
+                    "Content type is currently not supported by Apodini GRPC exporter. Use Protobuffers instead."
+                ))
+            }
+
+            var context = context
+
+            let promise = request.eventLoop.makePromise(of: Vapor.Response.self)
+            var lastMessage: GRPCMessage?
+            request.body.drain { (bodyStream: BodyStreamResult) in
+                switch bodyStream {
+                case let .buffer(byteBuffer):
+                    guard let data = byteBuffer.getData(at: byteBuffer.readerIndex, length: byteBuffer.readableBytes) else {
+                        return request.eventLoop.makeFailedFuture(GRPCError.payloadReadError("Cannot read byte-buffer from fragment"))
+                    }
+
+                    // retrieve all GRPC messages that were delivered in this request
+                    // (may be none, one or multiple)
+                    var messages = self.getMessages(from: data)
+                    // retain the last message, to run it through the handler
+                    // once the .end message was received.
+                    lastMessage = messages.popLast()
+
+                    messages
+                        // For now we only support
+                        // - one message delivered in one frame,
+                        // - multiple messages delivered in one frame.
+                        // One message delivered in multiple frames is not yet supported.
+                        // See `getMessages` internal comments for more details.
+                        .filter({ $0.isComplete })
```

Comment:
```suggestion
                        .filter(\.isComplete)
```

Swift supports [keypath expressions as functions](https://github.com/apple/swift-evolution/blob/53c2e80bdede03b99aee683fe6399b6e4cf0bf95/proposals/0249-key-path-literal-function-expressions.md)